### DISCUSSION TITLE
Intro App: Wizard Mode option

### DISCUSSION THREAD

**Discussion by mohamed2020m** [gsantner/markor#2079] [Ideas] [answered]
I think it's better to activate the Wizard Mode so that the user can clearly go back and forth between slides in the Intro App.

how to do it, so easy, just add the following line 

```java
setWizardMode(true);
```

inside the onCreate() callback in IntroActivity class.

here is the final result: 

![WizardMode2](https://github.com/gsantner/markor/assets/60801395/56389478-f710-4e85-bdae-e5c66e7ae74a) ![WizardMode ](https://github.com/gsantner/markor/assets/60801395/9cfc6bdc-e541-49c4-9694-da05c9d3cd8b) ![WizardMode3](https://github.com/gsantner/markor/assets/60801395/4b2a13b5-2bfb-48a0-abfb-7afe60252e28)

**Comment by gsantner** [comment, marked as answer]
@mohamed2020m Thanks you. Please feel free to make a merge request enabling it. Looks like it doesn't hurt 😃 .

**Reply by mohamed2020m** [reply]
@gsantner I'll do it.
